### ISSUE TITLE
Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift

### ISSUE THREAD

**Issue by dltmddbs100** [dltmddbs100/Paper-log#1]
## Main Idea
 **Improvement of training speed & stability using Batch Normalization**
 <https://arxiv.org/abs/1502.03167>
 ## Abstract
 Training Deep Neural Networks is complicated by the fact that the distribution of each layer’s inputs changes during
training, as the parameters of the previous layers change. This slows down the training by requiring lower learning
rates and careful parameter initialization, and makes it notoriously hard to train models with saturating nonlinearities. We refer to this phenomenon as internal covariate shift, and address the problem by normalizing layer inputs. Our method draws its strength from making normalization a part of the model architecture and performing the normalization for each training mini-batch. Batch Normalization allows us to use much higher learning rates and be less careful about initialization. It also acts as a regularizer, in some cases eliminating the need for Dropout. Applied to a state-of-the-art image classification model,
Batch Normalization achieves the same accuracy with 14 times fewer training steps, and beats the original model by a significant margin. Using an ensemble of batchnormalized networks, we improve upon the best published result on ImageNet classification: reaching 4.9% top-5 validation error (and 4.8% test error), exceeding the accuracy of human raters.

 ## Contribution 🤔
 + Internal Covariant Shift를 극복하기위해 Batch Noramlization이라는 방식을 제시
 + Activation function을 noramlization시키는 방식으로, Channel별 feature를 독립적으로 normalization 시킴으로써 input에서의 분포를 유지
 + Normalization을 통해 Internal Covariant Shift 를 해소, 그에 따라 발생하는 학습 속도 및 성능의 저하를 해결 
 + 높은 learning rate를 통한 안정적인 학습이 가능해짐에 따라 학습속도의 가속화
 + Initial parameter에 대한 gradients의 의존성 감소
 + 다음과 같은 조건에서 가장 효과적
 \- Increase learning rate
 \- Remove Dropout
 \- Reduce the L2 weight regularization
 \- Shuffle training examples more thoroughly
 ## Reviews
 <https://deepkerry.tistory.com/12>
